src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1323,17 +1329,37 @@ export class NetworkAreaDiagramViewer {
         }
     }
 
+    private getCssPropertyValue(displayedSize: number, propertyValue: ((value: number) => string) | string): string {
+        if (typeof propertyValue === 'function') {
+            return propertyValue(displayedSize);
+        }
+        return propertyValue;
+    }
```

Comment:
This function is not needed.
Just call directly the callbackFunction instead of using this getCssPropertyValue function.